view/frontend/web/template/shipping.html

```diff
@@ -59,7 +59,7 @@
 
                 <render args="shippingMethodListTemplate"/>
                 
-                <div class="reach-tax-duty" data-bind="fadeVisible: dutyCharges() > 0 && getReachEnabled() != 0">
+                <div class="reach-tax-duty" data-bind="fadeVisible: dutyCharges() > 0 && getReachEnabled() != 0 && getDHLEnabled() != 0">
```

Comment:
the only issue here is that the duties and taxes checkbox should show up if DHL is enabled, but it doesn't matter if Reach is enabled. They should work independently.

Reply:
(by "they" I mean the Reach vs DHL functionality)

Reply:
I do not know enough about the business rules to check/comment about the correctness here. So I would go with what Lizzy said.

I have to read/revisit the older meeting notes available in our system. But would appreciate if someone points me to any better source of information that they can think of/know of. Thanks.

Reply:
Updated code to reflect requested change.